mappings/oclcCatalog.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, source, namespace, constants):
         super(CatalogMapping, self).__init__(source, namespace, constants)
         self.mapping = self.createMapping()
 
+    def remove_oclc_prefixes(self, oclc_id):
+        return oclc_id.removeprefix('(OCoLC)').removeprefix('on').removeprefix('ocn').removeprefix('ocm')
```

Comment:
I recommend pulling up prefixes into a set or list above as a constant, e.g. OCLC_PREFIXES = ['(OCoLC)', 'on', 'ocn', 'ocm']. 

That way we can just do: 

```
for prefix in OCLC_PREFIXES:
  oclc_id = oclc_id.removeprefix(prefix)
```

In theory ha

Reply:
Updated!